deeppavlov/models/classifiers/keras_classification_model.py

```diff
@@ -162,7 +160,8 @@ def _change_not_fixed_params(self, **kwargs) -> None:
                 self.opt[param] = kwargs.get(param)
         return
 
-    def pad_texts(self, sentences: List[List[np.ndarray]]) -> np.ndarray:
+    def pad_texts(self, sentences: List[List[np.ndarray]],
+                  return_lengths=False) -> Union[np.ndarray, Tuple[np.ndarray, np.ndarray]]:
```

Comment:
```suggestion
                  return_lengths: bool = False) -> Union[np.ndarray, Tuple[np.ndarray, np.ndarray]]:
```
also, `return_lengths` is not described in docstring